util/src/bin/recurse.rs

```diff
@@ -52,30 +45,11 @@ struct Opts {
     #[clap(short = 't', long = "type", default_value = "A")]
     ty: RecordType,
 
-    /// Specify a nameserver to use, ip and port e.g. 8.8.8.8:53 or \[2001:4860:4860::8888\]:53 (port required)
-    /// ip:port are delimited by a comma like 8.8.8.8:53,1.1.1.1:53
+    /// Specify the IP addresses of the root zone nameservers to use.
+    ///
+    /// Multiple IP addresses may be delimited with commas.
     #[clap(short = 'n', long, use_value_delimiter = true, value_delimiter(','))]
-    nameserver: Vec<SocketAddr>,
-
-    /// Specify the IP address to connect from.
-    #[clap(long)]
-    bind: Option<IpAddr>,
-
-    /// Use ipv4 addresses only, default is both ipv4 and ipv6
-    #[clap(long)]
-    ipv4: bool,
-
-    /// Use ipv6 addresses only, default is both ipv4 and ipv6
-    #[clap(long)]
-    ipv6: bool,
-
-    /// Use only UDP, default to UDP and TCP
-    #[clap(long)]
-    udp: bool,
-
-    /// Use only TCP, default to UDP and TCP
-    #[clap(long)]
-    tcp: bool,
```

Comment:
I removed the `bind`, `udp`, and `tcp` options because they are no longer applicable with the simpler API. The `ipv4` and `ipv6` options only filtered addresses provided in `nameservers`, so they are not  needed.